amm/Makefile

```diff
@@ -31,15 +32,8 @@ check-amm: check-drink-tests ## Runs cargo (contract) check on AMM contracts.
 		cargo contract check --quiet --manifest-path $$d/Cargo.toml ; \
 	done
 
-.PHONY: build-test-contracts
-build-test-contracts: ## Builds contracts used in e2e-tests
-	@for d in $(TEST_PATHS); do \
-		echo "Building $$d contract" ; \
-		cd $$d && cargo contract build --quiet --release --features "contract" && cd $(SCRIPT_DIR) ; \
-	done
-
 .PHONY: build-all
-build-all: build-amm build-test-contracts ## Builds all contracts.
+build-all: build-amm ## Builds all contracts.
```

Comment:
Pedantic nit: How about removing `build-amm` phony and moving its contents to `build-all`? They are kinda synonyms now...

Reply:
Removed.